### DISCUSSION TITLE
Errors in console

### DISCUSSION THREAD

**Discussion by michelvermeulen** [intlify/bundle-tools#282] [Q&A]
Hi folks!

I'm using the vuei18n loader for Webpack as I can't loosen CSP policies in a Chrome extension. I have my strings stored in JSON files, and I've tried disabling the console errors like the one below but setting `warnHtmlMessage` to false in the browser action doesn't help. Any idea how I can get rid of those messages (outside of removing HTML formatting from strings)?

```
ERROR in ./src/common/locales/fr.json
Module Error (from ./node_modules/@intlify/vue-i18n-loader/lib/index.js):
(Emitted value instead of an instance of Error) [vue-i18n-loader]: /Users/------/locales/fr.json Detected HTML in 'THE STRING REDACTED' message.
 @ ./src/browser_action.js 10:0-42 14:6-8
```

**Comment by tatsumi-paypay** [comment]
@michelvermeulen were you able to find a solution?

**Reply by tatsumi-paypay** [reply]
I'm not sure if it will help, but I was able to fix that issue by fixing our `vue.config.cjs` file.
Here is the [gist](https://gist.github.com/tatsumi-paypay/f612c11fd8fda352e6ef38f2e0bfb30b)
